VegProcessor/hsi.py

```diff
@@ -586,6 +593,13 @@ def _calculate_static_vars(self):
             radius=self.testing_radius
         )
 
```

Comment:
Might be nice to have a log message here, i.e. `self._logger.info("calculating surround land etc etc)`